helpers/database.py

```diff
@@ -19,8 +19,9 @@ def get_where(self, **kwargs):
         """Returns a user that satisfies condition given by kwarg (e.g. zip_code='12345')"""
         assert len(kwargs) == 1
         match_k, match_v = list(kwargs.items())[0]
-        query = {'orderBy': match_k, 'equalTo': match_v}
-        return self.firebase.get('users', None, params=query.update(self.params)) or {}
+        query = {'orderBy': f'"{match_k}"', 'equalTo': f'"{match_v}"'}
```

Comment:
@mazore are you sure the query params need to be quoted like that? Are you able to test this with firebase? I don't have access to the account.

Reply:
Yes I’ve tested it out and this is how it works